in_toto/verifylib.py

```diff
@@ -300,34 +289,68 @@ def verify_all_steps_signatures(layout, chain_link_dict):
             }
 
   <Exceptions>
-    Raises an exception if a needed key can not be found in the passed
-    keys_dict or if a verification fails.
-    TBA (see https://github.com/in-toto/in-toto/issues/6)
+    ThresholdVerificationError
+            If any of the steps of the passed layout does not have enough
+            (`step.threshold`) links signed by different authorized
+            functionaries.
+
+  <Returns>
+    A chain_link_dict containing only links with valid signatures created by
```

Comment:
Very minor point: link_chain suggests a chain (made of links) and chain_link suggests a link (for a chain). Since this is a chain, I'd go with the former. (If it seems to make sense to people, don't worry about it. /: )

Reply:
I remember the discussion about the names for these dictionaries (c.f. https://github.com/in-toto/in-toto/pull/69#discussion_r100401016). I am definitely not happy with the current names, but I'm not convinced that `link_chain` is a huge improvement either. Given that the dicts are fairly well described in the docstring args sections (with examples), I guess we can leave them for the time being.